lib/dat-manager.js

```diff
@@ -19,18 +18,22 @@ module.exports = class Manager extends EventEmitter {
     })
   }
   _watch (dat) {
-    dat.on('update', () => this.emit('update'))
+    dat.events.on('update', () => this.emit('update'))
     return dat
   }
   get () {
     return this.dats
   }
   create (dir, cb) {
-    const dat = Dat({ dir })
-    dat.owner = true
-    dat.dir = dir
-    dat.share(err => {
+    Dat(dir, (err, dat) => {
       if (err) return cb(err)
+
+      dat.dir = dir
```

Comment:
👍  the only difference is `dat.path` will be fully resolved, not sure if `dir` was there.